src/iuse_actor.cpp

```diff
@@ -1854,12 +1870,18 @@ long musical_instrument_actor::use( player &p, item &it, bool t, const tripoint&
     /** @EFFECT_PER increases morale bonus when playing an instrument */
     const int morale_effect = fun + fun_bonus * p.per_cur;
     if( morale_effect >= 0 && calendar::turn.once_every( description_frequency ) ) {
-        if( !descriptions.empty() ) {
+        if( !descriptions.empty() && p.is_player() ) {
             desc = _( random_entry( descriptions ).c_str() );
+        } else if (!descriptions.empty() && p.is_npc() ) {
+            desc = string_format(_("%1$s %2$s"), p.disp_name(false).c_str(), random_entry( descriptions ).c_str() );
```

Comment:
This works in English, but not so much in other languages.
That earlier approach with separate field for NPC entries may be needed, except this time with those entries actually provided.

Reply:
Or add comments for translator. Check the teleporter sound for example.